frontends/web/src/components/groupedaccountselector/groupedaccountselector.tsx

```diff
@@ -142,6 +145,28 @@ export const GroupedAccountSelector = ({ title, disabled, selected, onChange, on
     return null;
   }
 
+  if (isMobile) {
+    return (
+      <>
+        <h1 className="title text-center">{title}</h1>
```

Comment:
I know @thisconnect , you're not a big fan of this. Can also use css module here. :)